src/Resources/config/menu.php

```diff
@@ -33,6 +33,7 @@
             ])
 
         ->set('sonata.admin.sidebar_menu', MenuItem::class)
+            ->share(false)
```

Comment:
if we cant add a test, we should at least leave a comment about why we do this, imho

Reply:
> > Otherwise menu is created once and then re-used - and that doesn't work if there is dynamic content in the menu.
> 
> @Warxcell Is there a way to add a test about the use-case which didn't work ?

Sure. Adding it atm.

Reply:
Added. :)